ci/Jenkinsfile

```
@@ -175,19 +175,21 @@ pipeline {
                         when {
                             expression { return caseList.contains(Case) }
                         }
+                        failFast false
```

Does this actually need to be set again?

---

Unresolving, since there was no response on whether this is really needed.

---

Not sure, I will have to test that using **Replay**

---

Ok I looked at the Pipeline for a bit a remembered why I had failFast in two places like that.  We needed failFast to support the new behavior for setting the fail label for when there there is a failure without a post section after the main pipeline.  Before having a global fail fast simply went to post.  Now the fine grain failFast is in each parallel section, one for Create Experiment and the other for Run Experiment.  If any one element of each parallel section fails the Failed label is set and the user in signaled that CI as not passed.